client/public/manifest.json

```diff
@@ -3,14 +3,9 @@
   "name": "Create React App Sample",
   "icons": [
     {
-      "src": "favicon.ico",
+      "src": "focus-logo.svg",
```

Comment:
ahhh I'm so sorry this is my fault. I should have been more clear. I would like to use the focus icon (which is the target) instead of the logo. I think this will look better because the whole focus logo in the browser is quite hard to read 
![image](https://user-images.githubusercontent.com/43567020/112392365-214bf380-8d5e-11eb-800e-a46ae4210307.png)